inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Similary here, the function name standiard_deviation  seems to fit better

Reply:
Furthermore the output data is not consistent with the other functions in this model. Change the output data to an array instead of a dictionary.